src/rtScriptV8/jsCallback.cpp

```diff
@@ -145,7 +145,11 @@ rtValue jsCallback::run()
   rtValue returnValue;
   if (tryCatch.HasCaught())
   {
+    #if defined RTSCRIPT_SUPPORT_V8
```

Comment:
Do all of the changes in each source file require a RTSCRIPT_SUPPORT_V8 or can they be generically applied to both v8 and node builds?  Assuming that Spark will always be built with node version 6.9.0 or later, if this change can be applied generically for both v8 and node then let's avoid all RTSCRIPT_SUPPORT_V8 checks and make a generic update.